metabrainz/templates/index/datasets.html

```diff
@@ -0,0 +1,86 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ _('Datasets') }} - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+  <h1 class="page-title">{{ _('The MetaBrainz Datasets') }}</h1>
+
+  <p> We ask <a href="{{ url_for('users.account_type') }}">commercial users</a> to <a href="{{ url_for('users.account_type') }}">support us</a> in order to help fund the creation and maintenance of these datasets. </p>
```

Comment:
This should be part of the next paragraph that starts `The MetaBrainz Foundataion provides free, open access...`